index.html

```diff
@@ -11063,10 +11063,10 @@ <h2>Index</h2>
  <dt>Selenium
  <dd>The following functions are defined within
   the <a href=https://selenium.dev>Selenium</a> project, at
-  revision <code>1721e627e3b5ab90a06e82df1b088a33a8d11c20</code>.
+  revision <code>a6b161a159c3d581b130f03a2e6e35f577f38dec</code>.
   <ul>
-   <!-- bot.dom.getVisibleText --> <li><dfn><a href=https://github.com/SeleniumHQ/selenium/blob/a6b161a159c3d581b130f03a2e6e35f577f38dec/javascript/atoms/dom.js#L1007><code>bot.dom.getVisibleText</code></a></dfn>
-   <!-- bot.dom.isShown --> <li><dfn><a href=https://github.com/SeleniumHQ/selenium/blob/a6b161a159c3d581b130f03a2e6e35f577f38dec/javascript/atoms/dom.js#L573><code>bot.dom.isShown</code></a></dfn>
+    <!-- bot.dom.getVisibleText --> <li><dfn><a href=https://github.com/SeleniumHQ/selenium/blob/a6b161a159c3d581b130f03a2e6e35f577f38dec/javascript/atoms/dom.js#L1007><code>bot.dom.getVisibleText</code></a></dfn>
```

Comment:
Are these line numbers still accurate?

Reply:
Yes, those are fine. We only missed to update the visible text.